mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -307,14 +316,112 @@ def _generate_target(self, cfg, joints_3d, joints_3d_visible):
 
         return target, target_weight
 
+    def _megvii_generate_target(self, cfg, joints_3d, joints_3d_visible,
+                                kernel):
+        """Generate the target heatmap via "Megvii" approach.
+
+        Args:
+            cfg (dict): data config
+            joints_3d: np.ndarray ([num_joints, 3])
+            joints_3d_visible: np.ndarray ([num_joints, 3])
+            kernel: Kernel of heatmap gaussian
+        Returns:
+            tuple: A tuple containing targets.
+
+            - target: Target heatmaps.
+            - target_weight: (1: visible, 0: invisible)
+        """
+
+        num_joints = cfg['num_joints']
+        image_size = cfg['image_size']
+        heatmap_size = cfg['heatmap_size']
```

Comment:
is heatmap size in `W, H` or `H, W`?

Reply:
how about use `W, H = cfg['heatmap_size']` and use W and H in the following

Reply:
It's W,H in the config file. And ok, I'll change heatmap_size to W, H.